src/components/custom-tree-item.jsx

```diff
@@ -53,6 +87,11 @@ const CustomContent = React.forwardRef(function CustomContent(props, ref) {
             <Typography onClick={handleSelectionClick} component="div" sx={styles.label}>
                 {label}
             </Typography>
+            {hover && (
+                <Box onClick={handleAddIconClick} sx={{ marginLeft: 'auto' }}>
```

Comment:
remove this `marginLeft: 'auto'` if you need to justify the button to the right please add `flexGrow: 1` to the preview container in the flexbox (here to put in styles.label)

Reply:
seems like it's already ok then why this marginLeft ?

Reply:
Indeed,I don't need the marginLeft.